.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
Also, what about running `npm run build:all` instead of calling bsb straight, that way if changes apply in the future it will handle those properly as well?